ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -871,7 +871,8 @@ then
 			if ver hwcmp PX4FMU_V5
 			then
 				set LOGGER_BUF 64
-				param set SDLOG_MODE 3
+				param set SDLOG_MODE 2
+				param set SDLOG_PROFILE 2
```

Comment:
No, idea, I was just trying not to change it, so I can set it to to the default profile if you think that is best.